### PR TITLE
Add Dependabot for GitHub Actions

### PR DESCRIPTION
**Pull request by Fokko**
I love dependabot <3

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#
```
Comment (zhjwpku) [inline .github/dependabot.yml]: It seems we don't have this leading `#` and the trailing `#` in other license blocks that use `#` as comment mark, should we keep it consistent?

Other than that, LGTM.

**Review by raulcd — APPROVED** [review]
LGTM, on some other projects we override (and increase) the PR limit to be 10 instead of the default 5:
https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#open-pull-requests-limit
Do we want to do so?

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
```
Comment (wgtmac) [inline .github/dependabot.yml:21]: For now it keeps an eye on the Github actions for update?

Comment (Fokko) [reply .github/dependabot.yml:21]: Exactly 👍

File: .github/dependabot.yml (hunk 1 above)
Comment (Fokko) [reply .github/dependabot.yml]: Good one, let's remove it 👍 
```suggestion
```

File: .github/dependabot.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    directory: "/"
+    schedule:
+      interval: "daily"
+
```
Comment (Fokko) [inline .github/dependabot.yml:26]:
```suggestion
    open-pull-requests-limit: 10

```

**Review by zhjwpku — APPROVED** [review]

**Review by raulcd — APPROVED** [review]
Thanks @Fokko

**Review by nastra — APPROVED** [review]

**Pull request merged**